t/http_proxy_connect.t

```diff
@@ -120,6 +120,65 @@ $t->stop();
 
 ###############################################################################
 
+$t->write_file_expand('nginx.conf', <<'EOF');
+
+%%TEST_GLOBALS%%
+
+daemon         off;
+
+events {
+}
+
+http {
+    %%TEST_GLOBALS_HTTP%%
+
+    log_format connect '$remote_addr - $remote_user [$time_local] "$request" '
+                       '$status $body_bytes_sent var:$connect_host-$connect_port-$connect_addr';
+
+    access_log %%TESTDIR%%/connect.log connect;
+
+    server {
+        listen  8081;
+        listen  8082;
+        access_log off;
+        location / {
+            return 200 "hello $remote_addr $server_port\n";
+        }
+    }
+
+    server {
+        listen       127.0.0.1:8080;
+        server_name  localhost;
+
+        # forward proxy for CONNECT method
+
+        proxy_connect;
+        proxy_connect_allow all;
+
+        proxy_connect_connect_timeout 10s;
+        proxy_connect_read_timeout 10s;
+        proxy_connect_send_timeout 10s;
+        proxy_connect_send_lowat 0;
+
+        proxy_connect_address 127.0.0.1:8082;
```

Comment:
new test case added, to test whether proxy_connect_address without nginx variable can work as expected.